demos/showcase/src/app/app.component.html

```diff
@@ -134,7 +134,7 @@ <h2>Showcase</h2>
                 </pxb-info-list-item>
 
                 <pxb-info-list-item divider="full" [subtitle]="['Phase A', 'Phase B', 'Phase C']" [avatar]="true">
-                    <div title>Input Voltage</div>
+                    <div pxb-title>Input Voltage</div>
```

Comment:
nice catch, I actually noticed this in another branch and was going to request someone fix it